apollo-spaceport/Cargo.toml

```diff
@@ -14,10 +14,12 @@ clap = { version = "3.1.3", default-features = false, features = ["derive"] }
 flate2 = "1.0.22"
 prost = "0.9.0"
 prost-types = "0.9.0"
+prost-wkt-types = "0.3.1"
```

Comment:
why is this used? It brings in a lot of dependencies (now we have prost 0.9 and 0.10 at the same time again, along with chrono and a few others)

is it for these lines: https://github.com/apollographql/router/pull/898/files#diff-bcf6e368342d150c072b96be426d833dc62a1d69cdc89ad333296210824d47cbR46 ? What are they used for?
(and if it's for build.rs, why isn't the crate imported in build-dependencies?)

Reply:
Ahh, I think this can be removed. I found an alternative solution.
Nice catch.

Reply:
fixed in [5067b39](https://github.com/apollographql/router/pull/898/commits/5067b3923edcadc19edec7ac291671ac1fc0f607)

Reply:
@o0Ignition0o removed it in [5067b39](https://github.com/apollographql/router/pull/898/commits/5067b3923edcadc19edec7ac291671ac1fc0f607)